src/PerfView/SupportFiles/UsersGuide.htm

```diff
@@ -6335,6 +6335,119 @@ <h5> Known issues (in Windows Version 1803 or earlier) </h5>
+    </p>
+    <p>
+        <strong>What happens if you don't use /EnableEventsInContainers:</strong> You will still capture all kernel 
+        events (CPU sampling, context switches, etc.) for container processes, but you will miss user-mode events 
+        like .NET garbage collection events, JIT events, exception events, and any custom EventSource events from 
+        those processes.
+    </p>
+
+    <h6>Step 2a: Analyze While Container is Running (Optional)</h6>
+    <p>
+        If the container(s) containing the process(es) of interest are still running when you stop the trace, you 
+        can open and analyze the trace directly on the host node. PerfView will be able to find binaries that it 
+        needs both on the host and inside the running containers through the container's file system view.
+    </p>
+    <p>
+        This is the simplest analysis path since no additional steps are required—just open the trace in PerfView 
+        on the host node.
+    </p>
+
+    <h6>Step 2b: Prepare Trace for Offline Analysis (Optional)</h6>
+    <p>
+        If you need to analyze the trace after the container has been shut down, or if you want to copy the trace 
+        to another machine for analysis, you need to prepare the trace while the container is still accessible. 
+        This is done using the merge command with the <strong>/ImageIDsOnly</strong> option.
+    </p>
+    <p>
+        First, copy the trace file into the container:
+    </p>
+    <ul>
+        <li>kubectl cp MyContainerTrace.etl.zip my-namespace/my-pod:/app/MyContainerTrace.etl.zip</li>
+    </ul>
+    <p>
+        Then, inside the container, run the merge command to inject the necessary image identification data:
+    </p>
+    <ul>
+        <li>PerfViewCollect merge /ImageIDsOnly MyContainerTrace.etl.zip</li>
+    </ul>
+    <p>
+        <strong>What /ImageIDsOnly does:</strong> When you run merge with /ImageIDsOnly, PerfView reads through 
+        the trace and for each DLL that was loaded by processes in the trace, it looks up the DLL's unique 
+        identifier (signature/timestamp) and injects that information into the trace. This unique identifier 
+        is what allows PerfView to later download the correct PDB symbols from a symbol server. Without this 
+        information, PerfView cannot resolve method names for code in those DLLs.
+    </p>
+    <p>
+        <strong>What happens if you don't run merge with /ImageIDsOnly:</strong> If you skip this step and later 
+        try to analyze the trace on another machine after the container is gone, PerfView will be unable to find 
+        the symbol files for DLLs that were loaded inside the container. Your stack traces will show addresses 
+        like "0x7ffe12345678" instead of method names for those DLLs. The managed code (.NET) symbols may still 
+        work since they come from NGEN PDBs or crossgen2 metadata, but native code (including the runtime itself) 
```

Comment:
Jitted code will still work, but nothing else will that comes from a binary inside of the container.